examples/simple.py

```diff
@@ -2,10 +2,10 @@
 
 import aiorwlock
 
+rwlock = aiorwlock.RWLock()
```

Comment:
I would suggest not changing the examples. I think it's still preferable to make them in an async context, where it makes sense.

Reply:
Reverted examples and README in e8eada18d19aae69f246d71aaa43f8000a25633f.